src/fiatExchanges/SelectProvider.test.tsx

```diff
@@ -186,6 +186,7 @@ describe(SelectProviderScreen, () => {
         [PaymentMethod.MobileMoney]: false,
         [PaymentMethod.Coinbase]: false,
         [PaymentMethod.FiatConnectMobileMoney]: false,
+        [PaymentMethod.Airtime]: false,
```

Comment:
on line 219 of this file there is a test `shows the provider sections` that should be updated to add a check for airtime